tests/system/providers/google/cloud/dataproc/example_dataproc_gke.py

```diff
@@ -55,7 +55,9 @@
     "workload_identity_config": {
         "workload_pool": WORKLOAD_POOL,
     },
-    "initial_node_count": 1,
+    "node_pools": {
```

Comment:
I think this change will break the test

Reply:
@VladaZakharova tests passed without breaking.

Reply:
Did you run this test in gcp project? I can see that my DAG is failing with this change:
![Screenshot 2024-05-22 12 08 18](https://github.com/apache/airflow/assets/80038284/7777d3bb-596a-4f85-bffb-46e0d4d7cd88)

Reply:
Ah I see. i didn't run it in GCP project but ran the system tests that is run as part of CI.

Reply:
I have fixed the cluster config @VladaZakharova per the error stacktrace you shared. But I didn't verify it by running against a gcp project.

Reply:
Yeah - it's one of the reasons why system tests are not run by "casual" contributors why we need the dashboards exposed by Google team :)  - because it requires setup and gcp account to run them generally speaking so @VladaZakharova - if yuo could use some of the intra-google setup to see if this PR passses and resolve the comment and we should be good to merge.

Reply:
(And to be precise "casual" in the sense of not having all teh setup to run all the system tests specific for the Google provider)